gulpfile.js

```diff
@@ -215,25 +216,25 @@ function createSHA256File(filename, cb) {
 
 // Create stub installer that will then download all the requirements
 gulp.task('package-simple', function(cb) {
-  runSequence(['check-requirements', 'clean'], 'create-dist-win-dir', ['generate',
+  runSequence(['check-requirements', 'clean'], 'create-dist-win-dir', ['generate',
     'prepare-tools'], 'package', 'cleanup', cb);
-});
+});
 
   // Create bundled installer
 gulp.task('package-bundle', function(cb) {
-  runSequence(['check-requirements', 'clean'], 'create-dist-win-dir', ['generate',
+  runSequence(['check-requirements', 'clean'], 'create-dist-win-dir', ['generate',
     'prepare-tools'], 'prefetch', 'package', 'cleanup', cb);

```

Comment:
that is doable and should work good, because download is going to be done often for jbds bits only, probably